src/lib/lazy-load.js

```diff
@@ -1,18 +1,72 @@
 const lazyLoadingModifier = 'lazy-loading';
 const lazyLoadingImageClass = `n-image--${lazyLoadingModifier}`;
 const lazyLoadingWrapperClass = `n-image-wrapper--${lazyLoadingModifier}`;
+const performanceSupported = ('performance' in window && typeof window.performance.mark === 'function');
+
+const uid = () => (Date.now() * Math.random()).toString(16);
+
+const setUid = (img) => {
+	const id = uid();
+	img.setAttribute('data-uid', id);
+	return id;
+};
+
+const getUid = (img) => {
+	return img.dataset && img.dataset.uid ? img.dataset.uid : img.getAttribute('data-uid');
```

Comment:
e.g. IE01. Basically questioning why not go for the simpler `return img.getAttribute('data-uid');`? Works in all browsers, no need to worry about browser support